tests/extract_tests.py

```diff
@@ -0,0 +1,61 @@
+"""
+Tests for data extraction module.
+Input: raw HTML file
+Output: plaintext that contains information about the company - no HTML tags.
+"""
+import requests
+import logging
+from pathlib import Path
+import regex as re
+from extract import DataExtractor
+
+def exist_tags_in_plaintext(text):
+    """
+    Check if the plaintext contains any HTML tags
+    :param text: plaintext
+    :returns: boolean
+    """
+    tag_re = re.compile(r'</?\s*[a-z-][^>]*\s*>|(\&(?:[\w\d]+|#\d+|#x[a-f\d]+);)')
+    if tag_re.search(text) is None:
+        return False
+    return True
+
+def test_tags(address, save_results = False):
+    """
+    Requests a URL, then runs following tests:
+    - Check if tags exists in raw HTML; expected: True
+    - Check if no tags in unfiltered, extracted data; expected: False
+    - Check if no tags in filtered, extracted data; expected: False
+    - Check if no tags in filtered, paragraph only, extracted data; expected: False
+    """
+    r = requests.get(address)
+    extractor = DataExtractor()
+    extractor.create_soup_from_html(r.text)
```

Comment:
Why aren't you testing with the open_file function instead, as that both tests the create_soup_from_html function and the open_file function. Now you have a function that isn't tested.